llama_index/schema.py

```diff
@@ -377,7 +377,7 @@ class TextNode(BaseNode):
             "{value} placeholders."
         ),
     )
-    metadata_seperator: str = Field(
+    metadata_separator: str = Field(
```

Comment:
Is there a way to have backwards compatibility here? This will raise some confusing errors for anyone with older code modifying this attribute